core/kits/documents/tabs/settings-layout.php

```diff
@@ -52,9 +52,11 @@ protected function register_tab_controls() {
 		$this->add_responsive_control(
 			'container_width',
 			[
-				'label' => esc_html__( 'Content Width', 'elementor' ) . ' (px)',
+				'label' => esc_html__( 'Content Width', 'elementor' ),
 				'type' => Controls_Manager::SLIDER,
+				'size_units' => [ 'px', '%', 'vw', 'custom' ],
 				'default' => [
+					'unit' => 'px',
```

Comment:
Not required, the default unit is always 'px'. It is defined in `Control_Base_Units()` class.

Reply:
Done.